src/common/themes/default/defaults.ts

```diff
@@ -26,6 +26,17 @@ export const theme: ThemeOptions = {
     },
   },
   cards: {
+    search: {
+      width: 264,
```

Comment:
The constants I'm adding here are normally passed through `theme.spacing`, for example in simpleCard down there the width is 32 * spacing(8px) = 256px. 
I see that your height for example is not a multiple of 8 which is a problem to do what I'm saying, so I don't know what I'd do...

- If it's okay to make cards a few pixels bigger or smaller to match the 8px spacing grid, great.
- If graphically they are pixel perfect, I've calculated 394 = 49.25, so contributor.height would be 49.25, that's not a terribly ugly code...
- And if you don't agree, just leave it like this, it is fine.

Reply:
Using the spacing requires the whole application to be based on this spacing constant.
The contents of the cards should be then dynamically calculated - lines truncated, wrapped, etc.
Having hardcoded dimensions is more reliable at this point imo.